test/users/engin/compiler/parentheseslessRecConst.future

```diff
@@ -0,0 +1,6 @@
+bug: paranthesesless inner* constructor doesn't compile with --verify
```

Comment:
Also, see my earlier comment - I don't think we should allow parenthesesless calls on constructors (but we should check with the rest of the team to be certain), so the fact that we error is correct but the message should be user-facing and say that we don't allow parenthesesless calls to constructors